dashboard/app/views/levels/_single_multi.html.haml

```diff
@@ -21,7 +21,7 @@
   = render partial: 'levels/content', locals: {app: app, data: data, content_class: question_content_class, source_level: level, hide_header: !standalone}
 
   - localized_questions = level.localized_property(:questions)
-  - if localized_questions.count
+  - if localized_questions.any?
```

Comment:
🤦‍♂ this mistake would be harder to make if ruby and javascript would agree on whether `0` is falsy

Reply:
right?!